src/pages/resources/Resources.css

```diff
@@ -1,3 +1,59 @@
-.links {
-  display: block;
-}
\ No newline at end of file
+.resources_wrapper {
+  height: 100%;
+}
+.top {
+  text-align: center;
+  padding: 4rem;
+}
+.bottom {
+  width: 80%;
+  margin: 0 auto;
+}
+.resources_list {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 30px;
+  width: 100%;
+  padding: 2rem;
+}
+.resources_list li {
+  width: 30%;
+  padding: 10px;
+  border-radius: 20px;
+  box-shadow: 1px 0px 9px -2px #5b6372;
+}
+.resources_list li img {
+  height: 120px;
```

Comment:
Let's only constrain the width and leave height dynamic as not all logo's have the 14x12 ratio
```suggestion
```

Reply:
done